cpp/src/slice2swift/Gen.cpp

```diff
@@ -1463,16 +1457,12 @@ Gen::ObjectVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
         baseNames.push_back(fixIdent(getRelativeTypeString(*i, swiftModule)));
     }
 
-    //
-    // Check for swift:inherits metadata.
-    //
-    const StringList metadata = p->getMetadata();
-    static const string prefix = "swift:inherits:";
-    for (StringList::const_iterator q = metadata.begin(); q != metadata.end(); ++q)
+    // Check for 'swift:inherits' metadata.
+    for (const auto& metadata : p->getMetadata())
```

Comment:
Would be nice to be able to get a list of metadata for a directive, maybe we don't do this in many places?

Reply:
We could definitely add it in the future, but indeed we don't need that functionality in many places, although there are a couple spots.